compiler-rs/clients_schema_to_openapi/src/paths.rs

```diff
@@ -311,6 +313,25 @@ fn get_path_parameters(template: &str) -> Vec<&str> {
     result
 }
 
+fn split_summary_desc(desc: String) -> SplitDesc{
+    let mut parts = desc.split(['.','\n',':']);
+    let first_line = parts.next().unwrap_or_else(|| "");
```

Comment:
You can use `desc.split_once()` that will return `Option<&str, &str>` which can greatly simplify the logic when there's only one sentence.

Also nit (running `cargo clippy` will show it): in that case you can use `.unwrap_or("")`: you don't need the closure since it returns a constant value. `unwrap_or_else()` is only useful when the "else" is an expression that we want to evaluate only when needed.